pkg/cache/cache.go

```diff
@@ -78,11 +78,19 @@ type Informer interface {
 	// AddEventHandler adds an event handler to the shared informer using the shared informer's resync
 	// period.  Events to a single handler are delivered sequentially, but there is no coordination
 	// between different handlers.
-	AddEventHandler(handler toolscache.ResourceEventHandler)
+	// It returns a registration handle for the handler that can be used to remove
+	// the handler again.
+	AddEventHandler(handler toolscache.ResourceEventHandler) (toolscache.ResourceEventHandlerRegistration, error)
```

Comment:
@FillZpp can we maybe have to distinct PRs, one that just bumps the deps and a follow-up that changes the interfaces so that our release note tooling will be able to properly call out the EventHandler change?

Reply:
Sure, I can do this, but the first one will fail to run golangci-lint and some other tests. Is it ok?

Reply:
Maybe just use a tiny wrapper around the upstream `AddEventHandler` that just does `_, _ = AddEventHandler(handler)`?